hash/oric1_cass.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+<softwarelist name="oric1" description="Oric 1 cassettes">
+
+	<software name="xenon1">
+		<description>Xenon 1</description>
+		<year>1983</year>
+		<publisher>IJK Software</publisher>
+		<part name="cass" interface="oric1_cass">
+			<dataarea name="cass" size="37396">
+				<rom name="Xenon 1 (UK) (1983).tap" size="37396" crc="ac9342ef" sha1="1acc057f7ce7d8432c59c7f43231b7e46c0213fa"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="xenon1a1" cloneof="xenon1">
+		<description>Xenon 1 [a1]</description>
+		<year>1983</year>
+		<publisher>IJK Software</publisher>
+		<part name="cass" interface="oric1_cass">
+			<dataarea name="cass" size="37390">
+				<rom name="Xenon 1 (UK) (1983) [a1].tap" size="37390" crc="cb0ded12" sha1="7c3d6933498edcc4e5a6f28743c20334b7603729"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="xenon1a2" cloneof="xenon1" supported="no">
+		<description>Xenon 1 [a2]</description>
+		<year>1983</year>
+		<publisher>IJK Software</publisher>
+		<part name="cass" interface="oric1_cass">
+			<dataarea name="cass" size="37396">
+				<rom name="Xenon 1 (UK) (1983) [a2].tap" size="37396" crc="e8536524" sha1="de6dc28770b8e900a427ccdf5c6d315a2ca93c0b"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Can you please not copy markup from the file names into descriptions?  Please follow the conventions in other software lists for disambiguation text.

Reply:
Done. (I think)
Not sure what software name to use for say Xenon 1 (Alt 1)